app/environments/brassbirmingham/brassbirmingham/envs/classes/roads/railroad.py

```diff
@@ -0,0 +1,10 @@
+from classes.player import Player
+
+from .enums import RoadType
+from .road import Road
+
+
+class Railroad(Road):
+    def __init__(self, owner: Player):
+        super(Railroad, self).__init__(owner, RoadType.railroad)
+        # TODO cost
```

Comment:
Maybe `cost` for `Road`s could be a function that intakes if we're building 2 of them? In the case of Canals, it would only accept building one at a time and always return (3,0,0) while Railroads could return (5, 1, 0) and (15, 2, 1) depending on the variable. And the different tuple values are cost, coalCost, and beerCost?